packages/tests/esm-misc/src/entities/codegen/AuthorCodegen.ts

```diff
@@ -163,22 +163,66 @@ export abstract class AuthorCodegen extends BaseEntity<EntityManager, string> im
     return getField(this, "updatedAt");
   }
 
+  /**
+   * Partial update taking any subset of the entities fields.
```

Comment:
Appreciate the invitation to bikeshed! :-)

In that vein, can we add a `\n` after every 1st sentence of the jsdoc, like:

```
/**
 * This is the first sentence.
 *
 * Now more details here.
 */
```

Just an old habit that is engrained. :-)